### DISCUSSION TITLE
mas outdated not showing updates on Big Sur 11.2.2

### DISCUSSION THREAD

**Discussion by producerguyaz** [Homebrew/discussions#1013] [Getting started]
Total newbie to Homebrew, not a coder.

Installed Homebrew to be able to install App Store updates without using App Store.  I have 2 updates showing, but "mas outdated" shows no results on Big Sur 11.2.2 and the "softwareupdate" command also thinks there's nothing new.

How to fix this so I can get my updates outside App Store?

**Comment by SMillerDev** [comment]
I guess you're experiencing: https://github.com/mas-cli/mas/issues/306 nothing homebrew can do about it.

**Comment by producerguyaz** [comment]
ugh... thanks.
